src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -385,6 +393,20 @@ const setRecordValuesOnTemplate = (templateSections, fieldMappings, record) => {
     return sections;
 };
 
+/*******************************************************************************
+ * @description Method checks for page level access. Currently checks
+ * if Advanced Mapping is on from the Data Import Custom Settings and
+ * if the Gift Entry Feature Gate is turned on.
+ */
+const getPageAccess = async () => {
+    const dataImportSettings = await getDataImportSettings();
```

Comment:
I am not sure if I want to block this PR over this, but I wonder if in the future the logic here should be handled in an apex controller method - I'm only thinking of minimizing the number of calls we need to make to the server before we can render the page.

Reply:
I thought about it earlier on but we didn't really have a unified entry point for Gift Entry, now that we have that entry point `geHome` we can migrate it to be handled from an apex controller and check for access before while we're deciding which view to render. But also i'm not sure what plans we have for the views later on, will they ever become tabs of their own?

Reply:
I think they will all be under one tab for the forseeable future, but I think the current implementation is ok for this story - just something to think about for potential future refactor

Reply:
Just chime in, the first server trip we make in the Gift Entry tab happens in `TemplateBuilderService` it might be worth revisiting that component (rename it to `GiftEntryService`?) and include this call there. I think we might not even want to bother making the trip to `BDI_MappingServiceAdvanced` to collect our maps if these settings aren't on.

Reply:
@voduyemi - just chatted with @psadie-sfdo and we agreed that the broader refactor is out of scope for this story - in which case I would just leave this "getPageAccess" method as it currently is and we can approach later, unless you already have work in progress. Thanks!!